app/views/programming_groups/index.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 = ProgrammingGroup.model_name.human(count: 2)
+
+= render(layout: 'shared/form_filters') do |f|
+  .col-auto
+    = f.label(:exercise_id, t('activerecord.attributes.programming_group.exercise_title'), class: 'visually-hidden form-label')
+    = f.collection_select(:exercise_id, Exercise.with_programming_groups, :id, :title, class: 'form-control', prompt: t('activerecord.attributes.programming_group.exercise_title'))
+
+.table-responsive
+  table.table.mt-4
+    thead
+      tr
+        th = t('activerecord.attributes.programming_group.programming_group_id')
+        th = t('activerecord.attributes.programming_group.exercise_title')
+        th = t('activerecord.attributes.programming_group.participants')
+        th = t('shared.actions')
```

Comment:
Other lists also feature some other nice columns. Two interesting ones I thought of might be the `created_at` or just a column with the number of users.